inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -310,12 +310,14 @@ def raise_if_match_workflow(obj, eng):
         skip_halted=True
     )
     if bool(matched_ids):
+        urls = ['https://inspirehep.net/holdingpen/{}'.format(id) for id in matched_ids]
```

Comment:
The link is hardcoded and it will not work for QA and beta. Instead you should do this:
https://github.com/inspirehep/inspire-next/blob/master/inspirehep/utils/url.py#L106

```suggestion
        urls = [ get_hep_url_for_recid(id, 'holdingpen') for id in matched_ids]
```